boilerplate/app/app.tsx.ejs

```diff
@@ -13,6 +13,16 @@ import { BackButtonHandler } from "./navigation/back-button-handler"
 import { contains } from "ramda"
 import { DEFAULT_NAVIGATION_CONFIG } from "./navigation/navigation-config"
 
+/**
+ * Ignore some yellowbox warnings. Some of these are for deprecated functions
+ * that we haven't gotten around to replacing yet.
+ */
+ YellowBox.ignoreWarnings([
+  "Async Storage has been extracted",
+  "componentWillMount is deprecated",
+  "componentWillReceiveProps is deprecated",
+])
```

Comment:
Good idea!

Reply:
To be clear: I want to resolve these soon - I just didn't feel the need to do it in this PR.

Reply:
Isn't the first one coming from Story book: https://github.com/storybookjs/storybook/issues/6078 ?